include/directinput/Manager.h

```diff
@@ -28,9 +28,12 @@ namespace Framework
 
 			void					CreateKeyboard();
 			void					CreateJoysticks();
+			std::list<GUID>			GetDevicesIds();

```

Comment:
Probably should rename this to `GetJoystickIds` since keyboard can technically be a part of the complete list of devices.